java/client/src/main/java/glide/api/commands/SortedSetCommands.java

```diff
@@ -0,0 +1,87 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.commands;
+
+import glide.api.models.commands.ZaddOptions;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+
+/**
+ * Sorted set Commands interface.
+ *
+ * @see <a href="https://redis.io/commands/?group=sorted-set">Sorted set Commands</a>
+ */
+public interface SortedSetCommands {
```

Comment:
Please rename to `SortedSetBaseCommands` since these commands go into the `BaseClient`.